mail_multicompany_force_company/models/ir_mail_server.py

```diff
@@ -0,0 +1,21 @@
+from odoo import api, models
+from odoo.osv import expression
+
+
+class IrMailServer(models.Model):
+    _inherit = "ir.mail_server"
+
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
```

Comment:
You don't actually use these named parameters, so you can replace them by `**kwargs`, also in the `super` call